### DISCUSSION TITLE
Feature Request: Sync starred talks between browsers

### DISCUSSION THREAD

**Discussion by tyll** [pretalx/pretalx#1786] [Ideas] [answered]
I starred talks at https://pretalx.com/devconf-cz-2024/schedule/ on my laptop while I am logged in. After I logged into pretalx on my mobile phone, I was expecting the stars to be synced so that I can filter for them while looking at my phone at the conference. Please support syncing the talks between sessions.

**Comment by rixx** [comment]
Stars are currently held entirely client-side, to allow non-logged-in users (most attendees) to star talks. Sharing / syncing will probably rely on an export/import system once #1002 is implemented.

**Comment by tyll** [comment]
It would be great to also solve starred talks are not synced between the different URLs for the schedule:

https://www.devconf.info/cz/schedule/ and https://pretalx.com/devconf-cz-2024/schedule/ show the schedule but they have different starred talks. The former gets the schedule via https://pretalx.com/devconf-cz-2024/widgets/schedule.js

**Reply by rixx** [reply]
In the frame of this feature request, that would require us to offer a login mechanic within the schedule widget. I don't anticipate that being a priority. However, the aforementioned export/import system could be used here, too.

**Comment by rixx** [comment, marked as answer]
Favourited talks are now saved to user accounts, and users are also shown a one-time indication if their favs will be stored only locally or synced with the server, in order to make them aware of the option to log in / create an account for this purpose.
